api/v1alpha4/virtualmachineclassinstance_types.go

```diff
@@ -0,0 +1,51 @@
+// © Broadcom. All Rights Reserved.
+// The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1alpha4
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// VirtualMachineClassInstanceSpec defines the desired state of VirtualMachineClassInstance.
+// It is a composite of VirtualMachineClassSpec.
+type VirtualMachineClassInstanceSpec struct {
+	VirtualMachineClassSpec `json:",inline"`
+}
+
+// VirtualMachineClassInstanceStatus defines the observed state of VirtualMachineClassInstance.
+type VirtualMachineClassInstanceStatus struct {
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Namespaced,shortName=vmclassinstance
+// +kubebuilder:storageversion
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="CPU",type="string",JSONPath=".spec.hardware.cpus"
+// +kubebuilder:printcolumn:name="Memory",type="string",JSONPath=".spec.hardware.memory"
+// +kubebuilder:printcolumn:name="Capabilities",type="string",priority=1,JSONPath=".status.capabilities"
```

Comment:
VM classes don't have capabilities _yet_.  So this can be removed.  If you want, you can also remove it from the virtualmachineclass_types.go